api/envoy/config/filter/network/tcp_proxy/v2/tcp_proxy.proto

```diff
@@ -134,46 +119,8 @@ message TcpProxy {
     repeated TCPRoute routes = 1 [(validate.rules).repeated .min_items = 1];
   }
 
-  // TCP Proxy filter configuration using deprecated V1 format. This is required for complex
-  // routing until filter chain matching in the listener is implemented.
-  //
-  // Example:
-  //
-  // .. code-block:: yaml
-  //
-  //     - name: "envoy.tcp_proxy"
-  //       config:
-  //         deprecated_v1: true
-  //         value:
-  //           stat_prefix: "prefix"
-  //           access_log:
-  //             - ...
-  //           route_config:
-  //             routes:
-  //               - cluster: "cluster"
-  //                 destination_ip_list:
-  //                   - "10.1.0.0/8"
-  //                 destination_ports: "8080"
-  //                 source_ip_list:
-  //                   - "10.1.0.0/16"
-  //                   - "2001:db8::/32"
-  //                 source_ports: "8000,9000-9999"
-  //
-  // .. attention::
-  //
-  //   Using the deprecated V1 configuration excludes the use of any V2 configuration options. Only
-  //   the V1 configuration is used. All available fields are shown in the example, although the
-  //   access log configuration is omitted for simplicity. The access log configuration uses the
-  //   :repo:`deprecated V1 access log configuration<source/common/json/config_schemas.cc>`.
-  //
-  // .. attention::
-  //
-  //   In the deprecated V1 configuration, source and destination CIDR ranges are specified as a
-  //   list of strings with each string in CIDR notation. Source and destination ports are
-  //   specified as single strings containing a comma-separated list of ports and/or port ranges.
-  //
-  // Deprecation pending https://github.com/envoyproxy/envoy/issues/4457
-  DeprecatedV1 deprecated_v1 = 6;
+  // [#not-implemented-hide:] Deprecated.
```

Comment:
Should `protodoc` have a `[#deprecated:]` annotation that does this? Probably easy to add.

Reply:
Yes that sounds good. I can work on this in a follow up if that is OK.

Reply:
Ack